huggingface/pytorch/inference/docker/1.10/py3/sdk1.18.0/Dockerfile.neuron

```diff
@@ -0,0 +1,139 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG MMS_VERSION=1.1.8
+ARG PYTHON=python3.7
+ARG PYTHON_VERSION=3.7.10
+# Pin torch neuron version to sdk1.17.1
+ARG TORCH_NEURON_VERSION=">=1.10.1,<1.10.2"
```

Comment:
Please change this to torch-neuron>=1.10.2,<1.10.3. For future please check the [following](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/releasecontent.html)  to find the right framework versions with the neuron sdk release.